api/v1/test_helpers.go

```diff
@@ -225,6 +225,120 @@ func CreateBaseYtsaurusResource(namespace string) *Ytsaurus {
 	return ytsaurus
 }
 
+func CreateBaseYtsaurusResourceWithMasterCaches(namespace string) *Ytsaurus {
```

Comment:
If only masterCaches are being added here, lets call CreateBaseYtsaurusResource and add mastercaches to spec without copy pasting its code.

Reply:
yes, will do it

Reply:
this is yet to be done

Reply:
looks like unresolved

Reply:
Sorry, forgot to remove it